pkgs/os-specific/windows/default.nix

```diff
@@ -0,0 +1,29 @@
+{ newScope, crossLibcStdenv }: let
```

Comment:
FWIW darwin's went in `pkgs/top-level/darwin-packages.nix` but used to be in `pkgs/os-specific/darwin/default.nix`. I'm not sure which I like better; I think @LnL7 made the switch?

Reply:
I prefer this way just for the shorter file paths. Also top-level is starting to get crowded.

Reply:
I moved `darwin = { ... };` from all-packages to a separate file, I don't think there was a default.nix. Almost every package set points to a `pkgs/top-level/<foo>-packages.nix` so I think that's better for consistency's sake.